app/src/main/res/values-es/strings.xml

```diff
@@ -1,98 +1,180 @@
-<?xml version="1.0" encoding="utf-8"?>
 <resources>
     <string name="app_name">UserLAnd</string>
     <string name="welcome">¡Bienvenido a UserLAnd!</string>
     <string name="permission_request_code">1000</string>
-    <string name="apps">Apps</string>
-    <string name="error_session_name">Tienes que proporcione un nombre de sesión.</string>
-    <string name="error_empty_field">¡Debes completar toda la forma!</string>
-    <string name="error_filesystem_name">Tienes que proporcione un nombre para el sistema de archivos.</string>
-    <string name="error_username">Tienes que proporcione un nombre de usuario.</string>
+
+    <!-- Labels -->
+    <string name="apps">Aplicaciones</string>
```

Comment:
We prefer just `Apps` here because this is a button label and there is not much room on the screen.